scripts/setup-adapters.sh

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+SCRIPTDIR=$(dirname "$0")
+source $SCRIPTDIR/helper-functions.sh
```

Comment:
shouldn't this be `setup-helper-functions.sh`? And how do the tests pass without this?

Reply:
Ok makes sense to me.